.github/workflows/pull_request.yml

```diff
@@ -15,13 +15,27 @@ jobs:
         cd MyPackage
         swift package init --type library
         swift build
+      linux_static_sdk_pre_build_command: |
+        mkdir MyPackage
+        cd MyPackage
+        swift package init --type library
+      enable_linux_static_sdk_build: true
+      # Wasm
+      wasm_sdk_pre_build_command: |
+        mkdir MyPackage
+        cd MyPackage
+        swift package init --type library
+      enable_wasm_sdk_build: true
       # Windows
       windows_build_command: |
         mkdir MyPackage
         cd MyPackage
         swift package init --type library
         swift build
       enable_windows_docker: true
+      # TODO: jflat - Remove after PR testing
+      linux_static_sdk_versions: "[\"6.0\", \"6.1\", \"nightly-main\", \"nightly-6.2\"]"
+      wasm_sdk_versions: "[\"nightly-main\", \"nightly-6.2\"]"
```

Comment:
Will remove these before merging so that only the default versions (nightly-6.2) are tested, but I wanted to get more extended coverage when testing this PR.

Reply:
Removed in https://github.com/swiftlang/github-workflows/pull/142/commits/85552f942bc521fb3ff77694035589342c9da09d